### ISSUE TITLE
[1.x] Add the possibility to disable `parallelPosting` for the `BlackBoxBoundedContext`

### ISSUE THREAD

**Issue by dmytro-kashcheiev** [SpineEventEngine/core-java#1485]
Currently it's impossible to disable `parallelPosting` for `BlackBoxBoundedContext` since it uses `SystemSettings.defaults()`. And by default for all environment except `io.spine.base.Tests` it will return `SystemSettings` with enabled parallel posting. As a result, you can't create **your own test environments** and use `BlackBoxBoundedContext`.

**Comment by armiol** [comment]
Here is how to disable the parallel posting for `BlackBoxContext` (at least, according to Spine 1.8.2 API):

```java
        BoundedContextBuilder builder = newBuilder();
        builder.add(new MyAggregateRepository())
               .add(...);
        builder.systemSettings().disableParallelPosting();
        BlackBoxContext ctx = BlackBoxContext.from(builder);
```

@dmytro-kashcheiev please let me know if that works for you.

**Comment by armiol** [comment]
Addressed for v1 branch via #1495.

@dmytro-kashcheiev as of `1.9.0-SNAPSHOT.5` the code above works properly. This fix will be available as a part of 1.9.0 release.
